src/Microsoft.DotNet.Interactive/Server/KernelCommandEnvelope.cs

```diff
@@ -45,7 +45,7 @@ protected KernelCommandEnvelope(KernelCommand command)
 
         KernelCommand IKernelCommandEnvelope.Command => _command;
 
-        internal static void RegisterCommandTypeForSerialization<T>() where T : KernelCommand
```

Comment:
This was a leftover from Ian pr, but no is useful (there is an equivalent counterpart on events) I am wondering if the registration of events / commands for serialization could be automated at write time. Yet this was useful in ensuring all commands and events were tested. we missed a couple :)

Reply:
How did we miss them, given the test that scans for them and makes sure they're all covered? Were they in other assemblies?